src/exodus/walletdb.h

```diff
@@ -0,0 +1,129 @@
+#ifndef EXODUS_WALLET_DB_H
```

Comment:
We should prefix with `ZCOIN`. Also `WALLET` and `DB` should reflect it file name (not a separated word).